newIDE/app/src/EventsFunctionsExtensionsLoader/MetadataDeclarationHelpers.js

```diff
@@ -906,6 +916,30 @@ const declarePropertyInstructionAndExpression = (
       .addParameter('yesorno', i18n._(t`New value to set`), '', false)
       .getCodeExtraInformation()
       .setFunctionName(setterName);
+
+    addObjectAndBehaviorParameters(
+      entityMetadata.addScopedAction(
+        toggleActionName,
+        i18n._(t`Toggle ${propertyLabel}`),
+        i18n._(
+          t`Toggle the property value for ${'"' + uncapitalizedLabel + '"'}.`
+        ) +
+          '\n' +
+          i18n._(
+            `If it was true, it will become false, and if it was false it will become true.`
+          ),
+        i18n._(
+          t`Toggle the property value for ${'"' +
+            uncapitalizedLabel +
+            '"'} of _PARAM0_`
```

Comment:
And here we can do shorter:

```suggestion
          t`Toggle property "${uncapitalizedLabel}" of _PARAM0_`
```